### DISCUSSION TITLE
Welcome to Algorithmist Discussions!

### DISCUSSION THREAD

**Discussion by NayakPenguin** [Algolisted-Org/AlgoListed#14] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NayakPenguin** [comment]
Please write  if you have any idea about the website

**Comment by ritiksaxena124** [comment]
Hey, the website is good but I have a suggestion the site logo and the logo at the footer don't match it would be great if both are the same

**Reply by NayakPenguin** [reply]
Great input, I will definitely look into it.
